### PR TITLE
Add create time to Trial API

### PR DESCRIPTION
**Pull request by andreyvelich**
Fixes: https://github.com/kubeflow/katib/issues/397.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/410)

### CODE REVIEW

File: pkg/api/api.proto [hunk 1]
```diff
@@ -366,6 +366,7 @@ message Trial {
     repeated Parameter parameter_set = 3; /// Hyperparameter set
     string objective_value = 4; /// Objective Value
     repeated Tag tags = 5; /// Tags of Trial.
+    string create_time = 6; /// Trial create timestamp RFC3339 format.
```
Comment (YujiOshima) [inline pkg/api/api.proto]: Please explain that the create_time will be filled in the server side even user set the value.

Comment (andreyvelich) [reply pkg/api/api.proto]: Done

**Pull request merged**